HW_7/task34.py

```diff
@@ -0,0 +1,29 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. Поскольку
+# разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам
+# стоит написать программу. Винни-Пух считает, что ритм есть, если число слогов (т.е. число
+# гласных букв) в каждой фразе стихотворения одинаковое. Фраза может состоять из одного
+# слова, если во фразе несколько слов, то они разделяются дефисами. Фразы отделяются друг
+# от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. В ответе
+# напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не
+# в порядке
+
+# Ввод:
+# пара-ра-рам рам-пам-папам па-ра-па-дам
+
+# Ввывод:
+# Парам пам-пам
+
+poem = input('Пиши стишок: ')
+# poem = 'пара-ра-рам рам-пам-папам па-ра-па-дам'
+poem = poem.lower()
+poem = poem.split()
+
+vowels = ['а', 'е', 'ё', 'и', 'о', 'у', 'ы', 'э', 'ю', 'я']
```

Comment:
Это можно задать просто строкой.
```suggestion
vowels = 'аяуюоеёэиы'
```